docs/reference/release-notes/highlights.asciidoc

```diff
@@ -24,18 +24,154 @@ Other versions:
+`search_after`] or the {ref}/point-in-time-api.html[PIT API documentation].
+
+[discrete]
+[[support-for-request-level-circuit-breakers]]
+=== Request-level circuit breakers on coordinating nodes
+
+You can now use a coordinating node to account for memory used to perform
+partial and final reduce of aggregations in the request circuit breaker. The
+search coordinator adds the memory that it used to save and reduce the results
+of shard aggregations in the request circuit breaker. Before any partial or
+final reduce, the memory needed to reduce the aggregations is estimated and a
+CircuitBreakingException is thrown if exceeds the maximum memory allowed in this
+breaker.
+
+This size is estimated as roughly 1.5 times the size of the serialized
+aggregations that need to be reduced. This estimation can be completely off for
+some aggregations but it is corrected with the real size after the reduce
+completes. If the reduce is successful, we update the circuit breaker to remove
+the size of the source aggregations and replace the estimation with the
+serialized size of the newly reduced result.
+
+[discrete]
+[[eql-case-sensitivity-operator]]
+=== EQL: Case-sensitivity and the `:` operator
 
-// Omit the notable highlights tag for entries that only need to appear in the ES ref:
-// [discrete]
-// === Heading
-//
-// Description.
+In 7.10, we made most EQL operators and functions case-sensitive by default.
+We've also added `:`, a new case-insensitive equal operator. Designed for
+security use cases, you can use the `:` operator to search for strings in
+Windows event logs and other event data containing a mix of letter cases.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search
+{
+  "query": """
+    process where process.executable : "c:\\windows\\system32\\cmd.exe"
+  """
+}
+----
+// TEST[setup:sec_logs]
+
+For more information, see the {ref}/eql-syntax-comparison-operators.html[EQL
+syntax documentation].
+
+[discrete]
+=== Deprecating REST API access to system indices
+
+We are deprecating REST API access to system indices. Most REST API requests
+that attempt to access system indices will return the following deprecation
+warning:
+
+[source,text]
+----
+this request accesses system indices: [.system_index_name], but in a future major version, direct access to system indices will be prevented by default
+----
+
+To allow projects external to {es} additional time to adjust their access
+patterns, these warnings are currently enabled for snapshot builds only.
+
+The following REST API endpoints access system indices as part of their
+implementation and will not return the deprecation warning:
+
+* `GET _cluster/health`
+* `GET {index}/_recovery`
+* `GET _cluster/allocation/explain`
+* `GET _cluster/state`
+* `POST _cluster/reroute`
+* `GET {index}/_stats`
+* `GET {index}/_segments`
+* `GET {index}/_shard_stores`
+* `GET _cat/[indices,aliases,health,recovery,shards,segments]`
```

Comment:
This feels more like a breaking change to me. @gwbrown Do you have any thoughts?

Reply:
While it doesn't actually "break" anything, that may be a better place to put it. My main concern is that it goes somewhere highly visible.

Reply:
I agree. I'll leave this here and add a related item to #63479.